src/components/Sidebar.js

```diff
@@ -1,26 +1,223 @@
-import React from 'react'
+import React from "react";
+import { useSelector } from "react-redux";
 
 const Sidebar = () => {
+  const isMenuOpen = useSelector((store) => store.toggle.isMenuOpen);
+  
+  const sidebarItems = [
+    {
+      title: "Home",
+      icon: "https://cdn-icons-png.flaticon.com/512/1946/1946488.png",
+      link: "/",
+    },
+    {
+      title: "Shorts",
+      icon: "https://cdn-icons-png.flaticon.com/512/2898/2898856.png",
+      link: "/",
+    },
+    {
+      title: "Subscriptions",
+      icon: "https://cdn-icons-png.flaticon.com/512/1946/1946488.png",
+      link: "/",
+    },
+    {
+      title: "Library",
+      icon: "https://cdn-icons-png.flaticon.com/512/2898/2898856.png",
+      link: "/",
+    },
+    {
+      title: "History",
+      icon: "https://cdn-icons-png.flaticon.com/512/1946/1946488.png",
+      link: "/",
+    },
+    {
+      title: "Your Videos",
+      icon: "https://cdn-icons-png.flaticon.com/512/2898/2898856.png",
+      link: "/",
+    },
+  ];
 
-  const sidebarItems = ["Home", "Shorts", "Subscriptions"]
-  const sidebarItems2 = ["Library", "History", "Your Videos", "Watch Later", "Liked Videos", "Show More"]
+  /* const sidebar = [{link : "Home", title : "Home", icon : ""}]
+  const sidebarItems2 = [
+    "Library",
+    "History",
+    "Your Videos",
+    "Watch Later",
+    "Liked Videos",
+    "Show More",
+  ]; */
+
+  if (!isMenuOpen) return null;
 
   return (
-    <div className='col-span-1 text-left m-2 p-2'>
-      <ul className='my-4'>
-        {
-          sidebarItems.map((sidebarItem) => <li className='p-2'>{sidebarItem}</li>)
-        }
+    <div className="col-span-1 text-left m-2 p-2">
+      <ul className="my-4">
+        {sidebarItems.map((sidebarItem, index) => (
+          <li key={index} className="p-2">
+            <div className="flex">
```

Comment:
use any one of justify-center items-center to center, according to use case
```suggestion
            <div className="flex justify-center items-center">
```

Reply:
not required to center sidebar items

Reply:
Please provide a screenshot how is it now?

Reply:
![image](https://user-images.githubusercontent.com/50284087/230398849-ba166e52-d4b9-4a40-bcf4-f02ed4904224.png)

Reply:
![image](https://user-images.githubusercontent.com/50284087/230401701-276c6530-bd3d-4773-89a6-94b9dd741f95.png)
Done